app/src/main/res/drawable-anydpi/ic_dark_mode.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:tint="?colorTertiary"
```

Comment:
Here is a warning from the CI, do you have any idea on how to correct it ?

Reply:
Good point, I missed that. It think I know why. It is due to the fact that colorTertiary is private theme attribute for some specific views (I don't know which ones). It can be used by all the views but maybe if a given view has not the colorTertiary attribute in its hidden structure, we have a warning. So I will use the color file instead of theme for this view.